src/common/logger.cpp

```diff
@@ -6,11 +6,20 @@
 
 #include "common/logger.hpp"
 
+#include <csignal>
 #include <cstdlib>
 
 #include "axom/slic.hpp"
 #include "mpi.h"
 
+namespace {
+void signalHandler(int signal)
+{
+  // Will clean up MPI via call to axom::Utilities::processAbort()
+  SLIC_ERROR("Received signal " << signal << ", exiting");
```

Comment:
I explicitly turned off SLIC's processAbort logic here:

https://github.com/LLNL/serac/blob/cb8c5295d72acb877029bc6c14a5342c33782be0/src/common/logger.cpp#L79

There has been a lot of complaints about how SLIC handles the aborting on parallel codes.  Did this exit?

Reply:
It didn't - I had misinterpreted the output.